os/board/rtl8730e/src/component/wifi/wifi_interactive_mode.c

```diff
@@ -883,7 +884,7 @@ static void cmd_debug(int argc, char **argv)
 	} else if (strncmp(argv[1], "is_up", strlen("is_up")) == 0) {
 		nvdbg("\r\n%d", wifi_is_up((rtw_interface_t)rtw_atoi((u8 *)argv[2])));
 	} else if (strncmp(argv[1], "set_mac", strlen("set_mac")) == 0) {
-		nvdbg("\r\n%d", wifi_set_mac_address(argv[2]));
+		//nvdbg("\r\n%d", wifi_set_mac_address(argv[2]));
```

Comment:
Could you let me know why you comment this out?

Reply:
It should be uncommented while raising the PR, I was commented to get, next error during compilation.